visitors/http_rest/normalize_arg_names.go

```diff
@@ -81,9 +81,11 @@ func (*argNameNormalizer) VisitDataChildren(self interface{}, c SpecVisitorConte
 }
 
 func (v *argNameNormalizer) setName(name ArgName) {
-	if _, ok := v.normalizationMap[name]; ok {
-		panic(fmt.Sprintf("Unexpected duplicated name for %v", name))
+	if existing, ok := v.normalizationMap[name]; ok {
+		glog.Warning("Non-normalized arg names ", existing, " and ", v.nonNormalizedArgName, " have conflicting normalized name ", name, ". Keeping the former and ignoring the latter.")
+		return
```

Comment:
Is this path only for conflicts? It looks like the warning may be inappropriate if there isn't any conflict, just the same name twice?

Reply:
The same name twice _is_ a conflict.